src/libs/filters/colors.c

```diff
@@ -54,6 +54,14 @@ static void _set_mask(dt_lib_filtering_rule_t *rule , const int mask, const gboo
 
 static gboolean _colors_clicked(GtkWidget *w, GdkEventButton *e, _widgets_colors_t *colors)
 {
+  // double click reset the widget
+  if(e->button == 1 && e->type == GDK_2BUTTON_PRESS)
+  {
+    _set_mask(colors->rule, 0, TRUE);
```

Comment:
_colors_update() does it anyway, but clearer to set the default as CL_AND_MASK (and).

Reply:
done